tests/model_generator/test_model_generator.py

```diff
@@ -44,7 +47,7 @@ class User(AvroModel):
     money_available: float
     name: typing.Optional[str] = None
     age: typing.Optional[types.Int32] = None
-    pet_age: types.Int32 = 1
+    pet_age: typing.Optional[types.Int32] = 1
```

Comment:
I think this is not correct: `An optional argument with a default does not require the Optional qualifier on its type annotation just because it is optional`

Reply:
The actual avsc schema is `["int", "null"]`, so this should be annotated as `int | None` (or, equivalently, `Optional[int]`)

Reply:
Ah yes, from Python point of view it is not required but we need it for the union in avro. Thanks